lib/nmatrix/math.rb

```diff
@@ -1045,4 +1086,53 @@ def dtype_for_floor_or_ceil
       self.__dense_map__ { |l| l.send(op,rhs) }
     end
   end
+
+  # Norm calculation methods
+  # Frobenius norm: the Euclidean norm of the matrix, treated as if it were a vector
+  def fro_norm
+    #float64 has to be used in any case, since nrm2 will not yield correct result for float32
```

Comment:
I honestly do not know. From the commit history at the other pull request (see #204) it seems that @MpoMp used nrm2 at some stage and changed it later but the reasoning wasn't documented. 

I'd be happy to look into it though. But it won't be tonight.